src/components/screens/reclaimBalance/modal/summary/index.js

```diff
@@ -31,23 +31,25 @@ const Summary = ({
     priorityOptions,
     transaction: {
       moduleAssetId,
-      nonce: account.info.LSK.sequence.nonce,
+      nonce: 0,
       senderPublicKey,
-      amount: account.info.LSK.legacy.balance,
+      amount: account.info.LSK.legacy?.balance,
     },
   });
 
   const onSubmit = async () => {
     const data = {
       network,
+      account: {
+        ...account.info.LSK,
+        sequence: { nonce: 0 },
+      },
       passphrase: account.passphrase,
-      account: account.info.LSK,
       transactionObject: {
         moduleAssetId,
-        senderPublicKey,
-        nonce: account.info.LSK.sequence.nonce,
         fee: toRawLsk(minFee.value),
-        amount: account.info.LSK.legacy.balance,
+        amount: account.info.LSK.legacy?.balance,
```

Comment:
The null check seems redundant. Aside from the fact that if `account.info.LSK.legacy` was `undefined` we wouldn't visit this page, the transaction fails if the amount if not defined. So we have to make sure `account.info.LSK.legacy.balance` if at this point defined and is a number.